playground/src/components/state-machines-details/state-machines-details.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import flowRight from 'lodash/flowRight';
-import { withApplicationContext } from '@commercetools-frontend/application-shell-connectors';
-import { connect } from 'react-redux';
+import { useDispatch } from 'react-redux';
+import { useApplicationContext } from '@commercetools-frontend/application-shell-connectors';
```

Comment:
Are these used in this component. Feels like I fail to spot that.

Reply:
Yes, for the `dataLocale`